components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -81,6 +81,10 @@ @interface MDCTabBarView ()
 /** The title font for bar items. */
 @property(nonnull, nonatomic, strong) NSMutableDictionary<NSNumber *, UIFont *> *stateToTitleFont;
 
+/** The content padding (as UIEdgeInsets) for each layout style. */
```

Comment:
suggestion: Should we mention that the NSNumber key is a wrapped MDCTabBarViewLayoutStyle and the NSValue wraps a UIEdgeInsets? Just for better context/readability

Reply:
I can do that. You don't think the generics types were sufficient to document the stored representation?

Reply:
I think it could help to be explicit for both. We mention UIEdgeInsets but not that the MDCTabBarViewLayoutStyle is wrapped.